docs/articles/new-expensify/workspaces/Create-expense-categories.md

```diff
@@ -0,0 +1,116 @@
+3. Tap **Delete category** to permanently delete the category.
+{% include end-option.html %}
+
+{% include end-selector.html %}
+
+# Enable or disable categories
+
+Once you have manually added your categories or automatically imported them from a connected accounting system, you can enable or disable the categories to determine whether they can be added to expenses. 
+
+{% include info.html %}
+After connecting an accounting system, Expensify automatically imports charts of accounts, GL accounts, expense accounts, and additional details into your workspace as **disabled** categories. Workspace admins can enable these categories to make them available for workspace members to add to their expenses.
+{% include end-info.html %}
+
+To enable or disable a category, 
+
+{% include selector.html values="desktop, mobile" %}
+
+{% include option.html value="desktop" %}
+1. Click your profile image or icon in the bottom left menu.
+2. Scroll down and click **Workspaces** in the left menu. 
+3. Select a workspace. 
+4. Click **Categories** in the left menu.
+5. Click a category and use the toggle to enable or disable it.
+
+{% include info.html %}
+You can enable, disable, or delete categories in bulk by selecting the checkbox to the left of the categories. Then click the “selected” dropdown menu at the top right of the page and select the desired option. 
+{% include end-info.html %}
+
+{% include end-option.html %}
+
+{% include option.html value="mobile" %}
+1. Tap your profile image or icon in the bottom menu.
+2. Tap **Workspaces**. 
+3. Select a workspace. 
+4. Tap **Categories**.
+5. Tap a category and use the toggle to enable or disable it. 
+
+{% include info.html %}
+You can enable, disable, or delete categories in bulk by selecting the checkbox to the left of the categories. Then tap the “selected” dropdown menu at the top of the page and select the desired option.
+{% include end-info.html %}
+
+{% include end-option.html %}
+
+{% include end-selector.html %} 
+
+# Automatic Expensify categories
+
+Over time, Expensify learns how you category specific merchants and automatically applies that category to the merchant in the future. 
+- If you change a category, Expensify learns that correction over time as well. However, changing a category on one expense does not change it for other expenses that have already been assigned the category.
+- Any expense rules for your workspace take priority over Expensify’s automatic categories. 
+- Expensify won’t automatically add a category to an expense if it is already manually assigned a category. 
+
+Expensify’s automatic categories are specific to each workspace. If you are part of multiple expenses, you’ll want to ensure the right workspace is active before creating an expense. Your active workspace is the most recently selected one. To change the active workspace, click your profile image or icon, then click Workspaces and select the correct workspace before you create the expense.
```

Comment:
```suggestion
Expensify’s automatic categories are specific to each workspace. If you are part of multiple workspaces, you’ll want to ensure the right workspace is active before creating an expense. Your active workspace is the most recently selected one. To change the active workspace, click your profile image or icon, then click Workspaces and select the correct workspace before you create the expense.
```

Also, I'm not sure these directions are accurate for NewDot?
> To change the active workspace, click your profile image or icon, then click Workspaces and select the correct workspace before you create the expense.

That's how it works for OldDot, but I don't _think_ we've implemented the same in NewDot. You just have to submit the expense to the correct workspace upon expense creation.

Reply:
Ah got it! I'll remove this copy